README.md

```diff
@@ -30,6 +30,11 @@ By default this plugin operates on source content files with the following file
 PELIMOJI_FILE_EXTENSIONS = ["md", "html", "rst", "txt"]
 ```
 
+`PELIMOJI_SIZE_MULTIPLIER` can be specified to increase the size of all emojis when they are rendered
```

Comment:
It could be helpful to add a bit more information for users:

1. The default value (`1`).
2. Somehow make it clearer that a value of `2` is two times larger than the default, `3` is 3x, etc.
3. Add a colon at the end (or a period if the setting description grows to more than one sentence).

Reply:
Just pushed an update to the wording; let me know if further tweaks are needed